packages/vite-plugin-checker/src/logger.ts

```diff
@@ -262,8 +230,8 @@ export function normalizeLspDiagnostic({
   fileText: string
 }): NormalizedDiagnostic {
   let level = DiagnosticLevel.Error
-  const loc = lspRange2Location(diagnostic.range)
-  const codeFrame = codeFrameColumns(fileText, loc)
+  const loc = tsLocationToBabelLocation(diagnostic.range)
```

Comment:
Since `typescript` and `vls` are both using `tsLocationToBabelLocation` now, it should be named. Maybe `tsLikeLocToBabelLoc` or something better.